### PR TITLE
:recycle: replace turbo by mocha

### PR DESCRIPTION
**Pull request by corinnekrych**
@jhellar Still WIP but not sure how to "mock" the acceptance test

### CODE REVIEW

**Comment by jcesarmobile** [comment]
@corinnekrych I think that tests aren't good, seems that they were copied from a hello world template but not adapted to this repo.
See what I did to adapt fh-connector-ibeacon-manager test https://github.com/danielpassos/fh-connector-ibeacon-manager/pull/1

**Comment by corinnekrych** [comment]
@jcesarmobile after discussing with @jhellar we agreed to removed those not useful
 acceptance tests

File: Gruntfile.js [hunk 1]
```diff
@@ -146,15 +125,10 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
 
   // Testing tasks
-  grunt.registerTask('test', ['jshint', 'shell:unit', 'shell:accept']);
-  grunt.registerTask('unit', ['jshint', 'shell:unit']);
-  grunt.registerTask('accept', ['env:local', 'shell:accept']);
+  grunt.registerTask('test', ['shell:unit']);
```
Comment (jhellar) [inline Gruntfile.js]: I would just add jshint: `...('test', ['jshint', 'shell:unit']);`

**Comment by corinnekrych** [comment]
@jhellar done as suggested, good to merge?

**Pull request merged**